core/arch/arm/plat-corstone1000/platform_config.h

```diff
@@ -0,0 +1,36 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2022, Arm Limited
+ */
+
+#ifndef PLATFORM_CONFIG_H
+#define PLATFORM_CONFIG_H
+
+#include <mm/generic_ram_layout.h>
+
+/* Make stacks aligned to data cache line length */
+#define STACK_ALIGNMENT		64
+
+#define GIC_BASE		0x1c000000
+
+#define UART0_BASE		0x1a510000
+#define UART1_BASE		0x1a520000
+#define CONSOLE_UART_BASE	UART1_BASE
+
+#define DRAM0_BASE		0x80000000
+#define DRAM0_SIZE		0x7f000000
```

Comment:
This is set twice (first time was with CFG_DDR_SIZE in the mk file).

So, please replace this line with:

#define DRAM0_SIZE		CFG_DDR_SIZE